projects/step-core/src/lib/modules/dynamic-forms/components/dynamic-field-group-editor/dynamic-field-group-editor.component.ts

```diff
@@ -313,4 +314,37 @@ export class DynamicFieldGroupEditorComponent implements OnChanges, OnDestroy {
       this.form.enable();
     }
   }
+
+  private isDeepEqual(valueA?: DynamicFieldGroupValue, valueB?: DynamicFieldGroupValue): boolean {
+    if (valueA === valueB) {
+      return true;
+    }
+    if ((valueA === undefined && valueB !== undefined) || (valueA !== undefined && valueB === undefined)) {
```

Comment:
Could be simplified to:
```suggestion
    if (valueA === undefined || valueB === undefined) {
```